app/chitter.rb

```diff
@@ -0,0 +1,89 @@
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative '../datamapper_setup.rb'
+require_relative './models/user'
+require_relative './models/peep'
+
+class App < Sinatra::Base
+
+  set :views, proc { File.join(root, '..', 'views') }
+  enable :sessions
+  set :session_secret, 'super secret'
+  register Sinatra::Flash
+  use Rack::MethodOverride
+
+  get '/' do
+    erb :index
+  end
+
+  get '/users/new' do
+    @user = User.new
+    erb :'users/new'
+  end
+
+  post '/users' do
+  	@user = User.create(email: params[:email], password: params[:password], password_confirmation: params[:password_confirmation], name: params[:name], username: params[:username])
```

Comment:
Line is too long. [179/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.